testers/python/server/markus_pyta_tester.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+from collections import defaultdict
+
+import python_ta
+from pylint.config import VALIDATORS
+from python_ta.reporters import PositionReporter, PlainReporter
+
+from markus_tester import MarkusTester, MarkusTest
+
+
+class MarkusPyTAReporter(PositionReporter):
+
+    def print_messages(self, level='all'):
+        # print to feedback file, then reset and generate data for annotations
```

Comment:
this should be a docstring

Reply:
Yeah, but for now I'm adding docstrings to the base classes only.